pkg/exporter/syslogexporter/README.md

```diff
@@ -7,7 +7,7 @@
 The syslog exporter supports sending messages to a remote syslog server.
 
 - This exporter can forward syslog messages to a third party [syslog server](https://www.rsyslog.com/) using RFC5424 and RFC3164
-- It also supports sending syslog messages to the [Cloud Syslog Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-syslog-source/) configured on a Sumo Logic hosted collector using the RFC5424 format
+- It also supports sending syslog messages to the [Cloud Syslog Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-syslog-source/) configured on a Sumo Logic hosted collector using the RFC5424 format, token required by Cloud Syslog Source can be added using logstransform processor, please see [example configuration](./examples/config_with_token.yaml)
```

Comment:
could we link `[logstransform processor]`?

Reply:
also RFC5424 -> `RFC5424`

Reply:
done :)  @sumo-drosiek please take a look once again